stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberElementUI.kt

```diff
@@ -167,6 +172,7 @@ fun PhoneNumberElementUI(
         placeholder = {
             Text(text = placeholder)
         },
+        interactionSource = interactionSource,
```

Comment:
What's the difference between this and `onValueChange`?

Reply:
The `MutableInteractionSource` is good for press, drag, and focus events but does not have support for tracking text input interactions so we need to manually track those interactions.